src/Display/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+
+import { Tabs, Props } from './Tabs';
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+
+export default {
+  title: 'Display/Tabs',
+  component: Tabs,
+  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
+  argTypes: {
+    className: {
+      table: {
+        disable: true,
+      },
+    },
+    activeTab: {
+      control: {
+        type: null,
+      },
+    },
+    onTabClick: {
+      control: {
+        type: null,
+      },
+    },
+  },
+} as Meta;
+
+const Template: Story<Props> = ({ ...args }) => (
+  <Tabs {...args}>
+    <Tabs.Pane tab="Location Tab">Tab Location</Tabs.Pane>
+    <Tabs.Pane tab="Salary">Tab Salary</Tabs.Pane>
+    <Tabs.Pane tab="Job">Tab Job</Tabs.Pane>
+    <Tabs.Pane tab="Company">Tab Company</Tabs.Pane>
+  </Tabs>
+);
+
+export const Interactive = Template.bind({});
+
+export const Variant = Template.bind({});
+Variant.args = {
+  variant: 'colored',
+};
```

Comment:
The current variant example seems to have the wrong style effect. When the tab is hovered, the whole tab will bound.  

https://www.loom.com/share/5e50bec973ff4eb2b9d63c946699441e

Reply:
Ooh, somehow the default for `theme` wasn't working.

Reply:
This should be solved.